src/sparseml/pytorch/image_classification/export.py

```diff
@@ -323,19 +323,52 @@ def export_setup(args_: ExportArgs) -> Tuple[Module, Optional[str], Any]:
 
     :param args_ : An ExportArgs object containing config for export task.
     """
-    save_dir, loggers = helpers.get_save_dir_and_loggers(args_, task=CURRENT_TASK)
-    input_shape = ModelRegistry.input_shape(args_.arch_key)
+    save_dir, loggers = helpers.get_save_dir_and_loggers(
+        task=CURRENT_TASK,
+        is_main_process=args_.is_main_process,
+        save_dir=args_.save_dir,
+        arch_key=args_.arch_key,
+        model_tag=args_.model_tag,
+        dataset_name=args_.dataset,
+    )
+    input_shape = ModelRegistry.input_shape(key=args_.arch_key)
     image_size = input_shape[1]  # assume shape [C, S, S] where S is the image size
-    (
-        train_dataset,
-        train_loader,
-        val_dataset,
-        val_loader,
-    ) = helpers.get_train_and_validation_loaders(args_, image_size, task=CURRENT_TASK)
+
+    val_dataset, val_loader = (
+        helpers.get_dataset_and_dataloader(
+            dataset_name=args_.dataset,
+            dataset_path=args_.dataset_path,
+            batch_size=1,
+            image_size=image_size,
+            dataset_kwargs=args_.dataset_kwargs,
+            training=False,
+            loader_num_workers=1,
+            loader_pin_memory=False,
+            max_samples=args_.num_samples,
+        )
+        if args_.is_main_process
+        else (None, None)
```

Comment:
export script is always the main process, correct?

Reply:
Yes! Good catch removing conditional in the next commit